contracts/test/MultiSigWalletDouble.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Utility Chain: MultiSigWallet
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+import "../MultiSigWallet.sol";
+
+
+contract MultiSigWalletDouble is MultiSigWallet {
```

Comment:
As it is used only for testing executeTransaction.So,add comment stating the intention.